openfl/federated/plan/plan.py

```diff
@@ -97,7 +97,7 @@ def parse(plan_config_path: Path, cols_config_path: Path = None,
                 defaults = plan.config[section].get(DEFAULTS)
 
                 if defaults is not None:
-                    defaults = WORKSPACE / 'workspace' / defaults
+                    defaults = join(plan_config_path.absolute().parent.parent, defaults)
```

Comment:
This path will work in the same way for CLI because `defaults` directory gets copied from `site-packages/openfl/openfl-workspace/workspace/plan/defaults`. But any change in copied files wouldn't affect the behavior because configuration files were parsed from the source `site-packages` directory. This change could be a solution to the problem if our new interactive API copied default config files as well. But it doesn't, so you can check that `openfl-tutorials/interactive_api_tutorials_(experimental)/Pytorch_Kvasir_UNET_workspace/new_python_api_UNET.ipynb` notebook is not working anymore. We are going to skip copying default configuration files during workspace creation to make things consistent, and this change will not be relevant anymore. This would probably lead to a misunderstanding of what files are actually taken for full plan generation because the workspace folder will no longer store files that `plan.yaml` default values point to.

Reply:
this change ensures that the yaml files which are actually stored in the 'local' folder structure are the ones that are used. It's extremely confusing otherwise since the main yaml file points to, for example, plan/defaults/aggregator.yaml, but any changes to that file locally get ignored in favor of the hidden 'global' aggregator.yaml.

It's convenient having access to all of the default yaml files locally for easier mental modeling of what is going to happen when the final file is constructed, but having them be secretly non-overwritable leads to a lot of wasted debugging time

Reply:
Hi @TortoiseHam! We discussed with the team internally that it would be better to provide the user with full plan specifications by copying a single `plan.yaml` file. It will not have any `default: /path/to/yaml` keys but will have all of the default files' settings. Thus, the user will know what can be overridden. It is going to be added in #116 where further discussion on this issue can be continued. In light of what we described, I believe this change should be reverted in this PR.

Reply:
alternatively, if you're going to be overhauling that system anyway, can you just take this pr and then apply whatever final decisions later?

Reply:
As I said in 1st comment to this change, applying it will lead to unstable behavior in develop branch due to broken interactive API tutorials where default Plan files are not copied to the workspace at all.

Reply:
alright, done